wd_core.au3

```diff
@@ -1503,7 +1494,10 @@ Func __WD_Delete($sURL)
 		_WinHttpCloseHandle($hOpen)
 	EndIf
 
-	Local $sMessage = (($sResponseText) ? ("ResponseText=" & StringLeft($sResponseText, $_WD_RESPONSE_TRIM) & "...") : (""))
+	__WD_ConsoleWrite($sFuncName & ": URL=" & $sURL, $_WD_DEBUG_Full)
```

Comment:
should be modified the same way like `;==>__WD_Get`

Reply:
Take a look on:
[__WD_MessageCreator](https://github.com/Danp2/au3WebDriver/pull/290/commits/99406453dd1c8852b82d61fdd9287cc78c4b482c)

Reply:
This conversation can be treated as resolved.